Doc/library/reprlib.rst

```diff
@@ -123,6 +123,16 @@ which format specific object types.
    similar manner as :attr:`maxstring`.  The default is ``20``.
 
 
+.. attribute:: Repr.indent
+
+   If this attribute is set to ``None`` (default), no indentation of the output
+   takes place. If it is set to either a string or an integer value, each
+   recursive representation will be on a new line, indented according to the
+   recursion depth. In that case, the attribute defines the string used for each
+   level of indentation. An integer value is equivalent to a string of space
+   characters of the same length.
```

Comment:
Here's a revised version of the above section attempting to make the text more clear, and adding examples to illustrate the behavior in an easily-accessible way.

```suggestion
   If this attribute is set to ``None`` (the default), the output is formatted
   with no line breaks or indentation, like the standard :func:`repr`.
   For example:

   .. code-block:: pycon

      >>> example = [
              1, 'spam', {'a': 2, 'b': 'spam eggs', 'c': {3: 4.5, 6: []}}, 'ham']
      >>> print(reprlib.repr(example))
      [1, 'spam', {'a': 2, 'b': 'spam eggs', 'c': {3: 4.5, 6: []}}, 'ham']

   If :attr:`~Repr.indent` is set to a string, each recursion level
   is placed on its own line, indented by that string:

   .. code-block:: pycon

      >>> reprlib.aRepr.indent = '-->'
      >>> print(reprlib.repr(example))
      [
      -->1,
      -->'spam',
      -->{
      -->-->'a': 2,
      -->-->'b': 'spam eggs',
      -->-->'c': {
      -->-->-->3: 4.5,
      -->-->-->6: [],
      -->-->},
      -->},
      -->'ham',
      ]

   Setting :attr:`~Repr.indent` to a positive integer value behaves as if it
   was set to a string with that number of spaces:

   .. code-block:: pycon

      >>> reprlib.aRepr.indent = 4
      >>> print(reprlib.repr(example))
      [
          1,
          'spam',
          {
              'a': 2,
              'b': 'spam eggs',
              'c': {
                  3: 4.5,
                  6: [],
              },
          },
          'ham',
      ]
```

Reply:
I am not sure we should handle negative numbers. It seems like a bit too much magic.

Reply:
Looks good to me. Thanks a lot! :) I'm always in favor of having short code examples but the current docs didn't have any, so I kept my addition in the same style.

Reply:
I have added you to the repository in case you didn't have writing permissions, yet. Feel free to commit your changes.

Reply:
@CAM-Gerlach I have added you to the repository in case you didn't have writing permissions, yet. Feel free to commit your changes.

Reply:
> I have added you to the repository in case you didn't have writing permissions, yet. Feel free to commit your changes.

Is there not a <kbd>Commit Suggestion</kbd> button below the suggestion above? You should be able to simply press that to apply my change directly; no need for all of that.

Reply:
@CAM-Gerlach I thought it won't show you as the commit author if I commit via <kbd>Commit Suggestion</kbd>. So I added the `Co-authored-by:` tag, but apparently GitHub adds that anyway. :)